custodial_copy.tf

```diff
@@ -27,29 +26,29 @@ resource "aws_iam_group_policy" "custodial_copy_group_policy" {
   policy = templatefile("${path.module}/templates/iam_policy/custodial_copy_policy.json.tpl", {
     account_id                 = data.aws_caller_identity.current.account_id
     secrets_manager_secret_arn = aws_secretsmanager_secret.preservica_secret.arn
-    entity_event_queue         = module.dr2_custodial_copy_notifications_queue.sqs_arn
-    database_builder_queue     = module.dr2_database_builder_queue.sqs_arn
+    custodial_copy_queue       = module.dr2_custodial_copy_notifications_queue.sqs_arn
+    database_builder_queue     = module.dr2_custodial_copy_db_builder_queue.sqs_arn
     management_account_id      = module.config.account_numbers["mgmt"]
   })
 }
 
-module "dr2_database_builder_queue" {
+module "dr2_custodial_copy_db_builder_queue" {
   source     = "git::https://github.com/nationalarchives/da-terraform-modules//sqs"
-  queue_name = local.custodial_copy_database_builder_queue_name
+  queue_name = local.custodial_copy_db_builder_queue_name
   sqs_policy = templatefile("./templates/sqs/sns_send_message_policy.json.tpl", {
     account_id = var.account_number,
-    queue_name = local.custodial_copy_database_builder_queue_name
+    queue_name = local.custodial_copy_db_builder_queue_name
     topic_arn  = local.custodial_copy_topic_arn
   })
   encryption_type = "sse"
```

Comment:
I forgot to ask, should we add this type to a variable in the `common.tf` file since it's used twice here and for the preingest?